pyne/origen22.py

```diff
@@ -1509,18 +1507,22 @@ def xslibs(nucs=NUCS, xscache=None, nlb=(201, 202, 203), verbose=False):
         t9[nlb[1]][field] = {}
     for field in FISSION_PRODUCT_FIELDS:
         t9[nlb[2]][field] = {}
-
     # fill with data
     for nuc in nucs:
-        if verbose:
-            print('computing {0}'.format(nucname.name(nuc)))
-        key = nucname.zzaaam(nuc)
-        if nuc in ACTIVATION_PRODUCT_NUCS:
-            _compute_xslib(nuc, key, t9[nlb[0]], xscache)
-        if nuc in ACTINIDE_AND_DAUGHTER_NUCS:
-            _compute_xslib(nuc, key, t9[nlb[1]], xscache)
-        if nuc in FISSION_PRODUCT_NUCS:
-            _compute_xslib(nuc, key, t9[nlb[2]], xscache)
+        try:
```

Comment:
Try blocks should be a couple of lines at most, and ideally should only be 1 line This is too all-encompassing.  I think you should have three try-except blocks, one in each of the conditionals.